README.md

```diff
@@ -91,6 +91,10 @@ To reload the config, you'll need to run `swaync-client --reload-config`
 - `keyboard-shortcuts`: `true` or `false`. If control center should use keyboard shortcuts
 - `image-visibility`: `always`, `when-available` or `never`. Notification image visiblilty
 - `transition-time`: uint (Any positive number, 0 to disable). The notification animation duration
+- `notification-center-margin-top`: uint (Any positive number, 0 to disable). The margin (in pixels) at the top of the notification center
+- `notification-center-margin-bottom`: uint (Any positive number, 0 to disable). The margin (in pixels) at the bottom of the notification center
+- `notification-center-margin-right`: uint (Any positive number, 0 to disable). The margin (in pixels) at the right of the notification center
+- `notification-center-margin-left`: uint (Any positive number, 0 to disable). The margin (in pixels) at the left of the notification center
```

Comment:
Doesn't only apply to centered control center. It would also be less confusing if it were to be called control-center-margin-right instead